packages/core/content-manager/admin/src/pages/EditView/utils/data.ts

```diff
@@ -262,6 +262,20 @@ const handleInvisibleAttributes = (
 
   for (const [attrName, attrDef] of Object.entries(schema.attributes)) {
     const fullPath = [...path, attrName].join('.');
+
+    // Skip processing fields with visible: false - they should be passed through as-is
+    // This function only handles conditions.visible (JSON Logic), not the visible boolean property
+    if ('visible' in attrDef && attrDef.visible === false) {
+      // Pass through the value as-is from data or initialValues
+      const userProvided = Object.prototype.hasOwnProperty.call(data, attrName);
```

Comment:
I think `Object.hasOwn` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn is the recommended method. It has good browser support now as far as I can tell. wdyt? It avoids prototype pollution edge cases and is more readable